packages/@dcl/inspector/src/components/Renderer/Renderer.tsx

```diff
@@ -89,10 +88,6 @@ const Renderer: React.FC = () => {
     const path = Object.keys(asset.contents).find(($) => isAsset($))
     let thumbnail: Uint8Array | undefined
 
-    if (!path && !asset.components[CoreComponents.MESH_RENDERER]) {
-      throw new Error('Invalid asset format: should contain at least one gltf/glb file or a mesh renderer component')
-    }
```

Comment:
This exception here is not necessary, it is from the times that the only kind of assets where gltfs, but with smart items there are some assets that have no models and they are valid items, like Text smart item.